rubocop/.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+inherit_gem:
+  eslint-config-umbrellio: lib/rubocop.yml
```

Comment:
Most reason is Rakefile, which can possibly by used for `.yml.erb`, if we will need it. I also use it for gemspec. And, finnaly, it allows to run `$ bundle exec rubocop --show-cop`